bindings/c/CMakeLists.txt

```diff
@@ -122,11 +122,20 @@ write_basic_package_version_file(
 )
 configure_file("${PROJECT_SOURCE_DIR}/cmake/Config.cmake.in" "${project_config}" @ONLY)
 
-fdb_install(
-  TARGETS fdb_c
-  EXPORT ${targets_export_name}
-  DESTINATION lib
-  COMPONENT clients)
+if(GENERATE_DEBUG_PACKAGES)
+  fdb_install(
+    TARGETS fdb_c
+    EXPORT ${targets_export_name}
+    DESTINATION lib
+    COMPONENT clients)
+else()
+  fdb_install(
+	TARGETS fdb_c
+    EXPORT ${targets_export_name}
+    DESTINATION lib
+    COMPONENT clients)
+endif()
```

Comment:
I'm still trying to figure out the right incantation to make this work, as it seems installing the file doesn't allow me to use the `EXPORT`, but the absence of the `EXPORT` breaks some other stuff. 

I'll also plan to fix the tab/space issue when I have a resolution to this.

Reply:
For now, I've removed this change from the PR, as the resolution is more complicated and the rest of the PR provides a significant win in package build times.